src/annotation.ts

```diff
@@ -285,7 +285,9 @@ export class Annotation extends Modifier {
     }
 
     L('Rendering annotation: ', this.text, x, y);
-    this.renderText(ctx, x, y);
+    this.x = x;
+    this.y = y;
```

Comment:
I see in the Element class there is a `setOrigin(x, y)`, but it looks surprisingly complicated. :-)

Do you know what the difference is between setOrigin(x,y) and what you're doing here?

There seems to be a x,y and a xShift,yShift and it is not clear what they each mean. We should clarify with comments in the Element class in a separate PR.

Reply:
Yes it is really complicated and only used by Articulation. It took a while to get it without visual differences with the x/yShift. Normally x and y is related to position and xShift and yShift with formatting (distance between different elements)